docker/daemon_unix.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/pkg/system"
 
-	_ "github.com/docker/docker/daemon/execdriver/lxc"
+	//"github.com/docker/docker/daemon/execdriver/lxc"
```

Comment:
This is similar to how [`database/sql`](https://golang.org/pkg/database/sql/) drivers usually work, too.